### DISCUSSION TITLE
Annotator Release Announcements and Calls for Translation

### DISCUSSION THREAD

**Discussion by phase1geo** [phase1geo/Annotator#13] [Show and tell]
# What is this discussion board about?

This discussion will act as an announcement board for upcoming Annotator releases and calls for translators to update any necessary translations to go into that release.

**If you are a Annotator translator or have interest in translating Annotator into your language, please follow/subscribe to this discussion thread.**

**Comment by phase1geo** [comment]
## Call for Annotator 1.0.0 translations!

I am planning on releasing the 1.0.0 release this Wednesday (9/8/2021).  Please have your translation pull requests submitted by 8pm (CST).

**Comment by phase1geo** [comment]
## Call for Annotator 1.1.0 translations!

I am planning on releasing the 1.1.0 release on Sunday (1/16/2022).  Please have your translation pull requests submitted by 8pm (CST).

**Comment by phase1geo** [comment]
## Call for Annotator 1.2.0 translations!

I am planning on releasing the 1.2.0 release on Friday (5/5/2023). Please have your translation pull requests submitted by 8pm (CST).

**Comment by phase1geo** [comment]
## Call for Annotator1.2.1  translations!

I am planning on releasing the 1.2.1 release on Monday (5/22/2023). Please have your translation pull requests submitted by 8pm (CST).

**Comment by phase1geo** [comment]
## Call for Annotator 2.0 translations!

I am planning on releasing the 2.0 release on Wednesday (10/30/2024). Please have your translation pull requests submitted by 8pm (CST).

**Comment by simonjoom** [comment]
Hello
I was looking for a editor project working with gtk4, and this is the best one that i found. (well i built gtk4 from source and i wanted to test it)

Also I was in need of a small utility to resize fast some picture, i use xfce and didn t wan t something big like gimp.


I ve some suggestion
I can just suggest to change icon to be a bit bigger, the ui is bit hard for beginners , but when we know all the keyboard shortcut it s cool. 
Also i did spent a while to find the export option, a menu on the right it s not usual for that (users use to have that on left side and it is also because the icons are very small)
Maybe some filter like image enhancer are not that hard to implement and could give more value and interests on the project,

i did made running this project under termux (android) i didn t find not much bug compare to shotwell ) 
Chatgpt helped me to Use glib-compile-schemas to generate the binary schemas.. maybe add that also in the INSTALL instructions 

thks you again
